README.md

```diff
@@ -35,6 +37,24 @@ on macOS/Linux
 
     `pip install fastapi mongodb uvicorn`
 
+
+## docker-compose between FastAPI and MongoDB
+
+Create docker-compose.yml file in the folder from which you launch your container.
```

Comment:
this line can be removed. File is part of repo

Reply:
I have done it